libs/wingsdk/src/target-sim/bucket.inflight.ts

```diff
@@ -67,6 +67,15 @@ export class Bucket implements IBucketClient, ISimulatorResourceInstance {
       .catch(() => false);
   }
 
+  public async exists(key: string): Promise<boolean> {
+    return this.context.withTrace({
+      message: `Exists (key=${key}).`,
+      activity: async () => {
+        return this.objectKeys.has(key);
```

Comment:
Can you please change this to return `this.fileExists(key)` and fileExists to return `this.objectKeys.has(key)`? this way both will keep the same behavior

Reply:
I decided to remove `this.fileExists(key)` because redundant. Instead, I replaced all calls to either `this.fileExists(key)` or `this.objectKeys.has(key)` with `this.exists(key)` so that everything is the same across the class and every call benefits from the `withTrace`.